src/pages/[...rest].ts

```diff
@@ -156,7 +155,7 @@ async function validateRef(name: string) {
 }
 
 type Platform = typeof PLATFORMS extends Set<infer T> ? T : never;
-const PLATFORMS = new Set(['idx', 'stackblitz', 'codesandbox', 'github', 'gitpod'] as const);
+const PLATFORMS = new Set(['firebase-studio', 'stackblitz', 'codesandbox', 'github'] as const);
```

Comment:
I wonder if we should temporarily keep support for the `?on=idx` query parameter. We’ll need to update use of it in the docs and keeping it for now would let us do that without breaking things temporarily:

https://github.com/withastro/docs/blob/4831e80343b948c9a62c753f44c40d76e6f9dec1/src/content/docs/en/tutorial/1-setup/index.mdx?plain=1#L33

I also looked for `?on=gitpod` and didn’t find anywhere _we_ use it, but I guess there may still be links somewhere?

Reply:
Yeah I wasn’t sure about the `gitpod` one. But if it’s broken anyway, then we can remove it. Maybe we should make it redirect to the astro.new homepage?